### ISSUE TITLE
User Story: Hosting more than one Lab on a Host

### ISSUE THREAD

**Issue by PlagueHO** [PlagueHO/LabBuilder#60]
Hosting more that one Lab on a Hyper-V host is possible, but because the VLAN's specified in the VM Switch configuration for each lab may collide, a User would need to adjust them so that there was no over lap.

Ideally a user would just install both labs and the VLANs assigned to the Lab Switches were adjusted to ensure Lab isolation.

The exception would be any External switches as they have to be shared by all Labs to provide internet access to the Lab machines.

**Comment by DragonReach** [comment]
As an aside, may also look at creating VMs on additional hosts as well for large lab scenarios - Most of the hyper-v cmdlets are remoteable so config is doable

**Comment by PlagueHO** [comment]
Definitely. That is a medium term goal and should be reasonably easy to implement.
In this respect it is also good that PS Direct stopped working. Some of the other issues I've raised are related to this (by ensuring each lab is isolated). We could also use Network Virtualization (instead of VLANs) if the hosts were Windows Server 2012 R2 and above I think (not on Windows 10 of course).
